homepage/models.py

```diff
@@ -48,11 +49,39 @@ def __str__(self):
 
 
 class AppUser(models.Model):
+    # This field is the built-in django user model
     user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
 
     def __str__(self):
         return self.user.username
 
+    def toggle_user_activation(self):
+        self.user.is_active = not(self.user.is_active)
+
+    @staticmethod
+    # Creates a new appUser (without superuser permmisions) and saves it in the DB
+    def create_appUser(username, email, password):
+        appUser = AppUser()
+        user = User.objects.create_user(username, email, password)
+        appUser.user = user
+

```

Comment:
I still think there should be a comment here explaining which fields from the django user we are using and for what

Reply:
i disagree
I think the code should have as few comments as possible, the code is rather simple and i don't see a section which needs further explaining with comments.

Is there a specific section you think could use more comments?

Reply:
in general - just short, one line comments explaining what each test is testing.
If it is testing valid input, if it is testing invalid input, etc.